packages/@react-aria/landmark/src/useLandmark.ts

```diff
@@ -233,17 +226,28 @@ class LandmarkManager {
 export function useLandmark(props: AriaLandmarkProps, ref: MutableRefObject<HTMLElement>): LandmarkAria {
   const {
     role,
+    tabIndex,
     'aria-label': ariaLabel,
     'aria-labelledby': ariaLabelledby
   } = props;
   let manager = LandmarkManager.getInstance();
   let label = ariaLabel || ariaLabelledby;
+  let [isLandmarkFocused, setIsLandmarkFocused] = useState(false);
+
+  let onFocusLandmark = () => setIsLandmarkFocused(true);
```

Comment:
given this is a custom event and we have nested landmarks, we'll want to stop propagation once a landmark has handled it